src/services/DocumentSelectionRequestProcessor.ts

```diff
@@ -65,6 +66,8 @@ export class DocumentSelectionRequestProcessor {
 			  const eventBody = JSON.parse(event.body);
 			  postOfficeSelection = eventBody.post_office_selection;
 			  selectedDocument = eventBody.document_selection.document_selected;
+				countryCode = eventBody.document_selection.country_code;
+				this.logger.info("DOCUMENT SELECTION BODY", eventBody.document_selection);
```

Comment:
Do we still want to keep this? if yes might be worth changing to 
this.logger.info({ message: "Document Selection Body" + JSON.stringify(eventBody.document_selection) });